flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/checkpointing/CheckpointBarrierHandler.java

```diff
@@ -118,6 +125,18 @@ public long getCheckpointStartDelayNanos() {
         return latestCheckpointStartDelayNanos;
     }
 
+    public long getBytesProcessedDuringAlignment() {
+        if (isDuringAlignment()) {
+            return bytesProcessedDuringAlignment;
+        } else {
+            return FutureUtils.getOrDefault(latestBytesProcessedDuringAlignment, 0L);
+        }
+    }
+
+    public long getCheckpointSyncDurationMillis() {
+        return latestCheckpointSyncDurationMillis;
```

Comment:
nit: rename both field and getter to `last/getLastCheckpointSyncDurationMillis`?